terraform/modules/happy-service-eks/variables.tf

```diff
@@ -198,6 +198,23 @@ variable "routing" {
   })
   description = "Routing configuration for the ingress"
 }
+
+variable "sidecars" {
+  type = map(object({
+    image : string
+    tag : string
+    port : optional(number, 80),
+    memory : optional(string, "100Mi")
+    cpu : optional(string, "100m")
+    image_pull_policy : optional(string, "IfNotPresent")
+    health_check_path : optional(string, "/")
+    initial_delay_seconds : optional(number, 30),
+    period_seconds : optional(number, 3),
```

Comment:
Generally, I don't think we should really expose these to our customers. I think leaving them here as optionals is fine as long as our examples keep them out and just use the defaults. I think these fields will rarely need to be different from the defaults and makes the configuration more complicated.

Reply:
I disagree. Sidecars are going to be 3rd party containers with non-standard behavior. Having the ability to tweak their healthchecks is essential, as a failing sidecar will prevent the main applicaiton to start. The cerbos sidecar we're deploying will have a non-standard healthcheck, so the set of options needed to be configured will only expand.

Reply:
The healthcheck one makes sense, I agree. I like the idea about tweaking them if need be, I just think we shouldn't advertise the other options. I think we'll get more questions about stuff like image_pull_policy and delay_seconds